'extends' is not compiling when used inside declaration file and importHelpers option is switched on

**TypeScript Version:**  2.1.4 / 2.2.0-dev.20161207

**Code**

```ts
interface Animal {
	type: string;
}

export declare class Pet implements Animal {
	public type: string;
	public name: string;
	constructor();
}

export declare class Dog extends Pet {
	public breed: string;
	constructor();
}
```

**Expected behavior:**
Compiles successfully.

**Actual behavior:**
Compilation fails with the following error:
```
error TS2354: This syntax requires an imported helper but module 'tslib' cannot be found.
```

The same source code compiles fine on TypeScript 2.1.1. Installed `tslib` package has version 1.2.0.

--- a/src/compiler/checker.ts
+++ b/src/compiler/checker.ts
@@ -17999,7 +17999,7 @@ namespace ts {
 
             const baseTypeNode = getClassExtendsHeritageClauseElement(node);
             if (baseTypeNode) {
-                if (languageVersion < ScriptTarget.ES2015) {
+                if (languageVersion < ScriptTarget.ES2015 && !isInAmbientContext(node)) {
                     checkExternalEmitHelpers(baseTypeNode.parent, ExternalEmitHelpers.Extends);
                 }
 

--- a/tests/baselines/reference/importHelpersDeclarations.symbols
+++ b/tests/baselines/reference/importHelpersDeclarations.symbols
@@ -1,0 +1,8 @@
+=== tests/cases/compiler/declaration.d.ts ===
+export declare class D {
+>D : Symbol(D, Decl(declaration.d.ts, 0, 0))
+}
+export declare class E extends D {
+>E : Symbol(E, Decl(declaration.d.ts, 1, 1))
+>D : Symbol(D, Decl(declaration.d.ts, 0, 0))
+}

--- a/tests/baselines/reference/importHelpersDeclarations.types
+++ b/tests/baselines/reference/importHelpersDeclarations.types
@@ -1,0 +1,8 @@
+=== tests/cases/compiler/declaration.d.ts ===
+export declare class D {
+>D : D
+}
+export declare class E extends D {
+>E : E
+>D : D
+}

--- a/tests/cases/compiler/importHelpersDeclarations.ts
+++ b/tests/cases/compiler/importHelpersDeclarations.ts
@@ -1,0 +1,9 @@
+// @importHelpers: true
+// @target: es5
+// @module: commonjs
+// @moduleResolution: classic
+// @filename: declaration.d.ts
+export declare class D {
+}
+export declare class E extends D {
+}

